tests/topotests/bgp_vrf_lite_ipv6_rtadv/test_bgp_vrf_lite_ipv6_rtadv.py

```diff
@@ -71,22 +71,6 @@ def setup_module(mod):
     router_list = tgen.routers()
 
     logger.info("Testing with VRF Lite support")
-    krel = platform.release()
-
-    # May need to adjust handling of vrf traffic depending on kernel version
-    l3mdev_accept = 0
-    if (
-        topotest.version_cmp(krel, "4.15") >= 0
-        and topotest.version_cmp(krel, "4.18") <= 0
-    ):
-        l3mdev_accept = 1
-
-    if topotest.version_cmp(krel, "5.0") >= 0:
-        l3mdev_accept = 1
-
-    logger.info(
```

Comment:
that commit log introduces udp_l3mdev and seems to modify ipv6 equivalent.
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=63a6fff353d0
I think this check is necessary, though the kernel version should be adapted.